packages/fx-core/src/question/inputs/CreateProjectInputs.ts

```diff
@@ -14,12 +14,7 @@ export interface CreateProjectInputs extends Inputs {
   /** @description Teams Toolkit: select runtime for your app */
   runtime?: "node" | "dotnet";
   /** @description New Project */
-  "project-type"?:
```

Comment:
Suggest not change this file manually, which is generated automatically.

Reply:
Can apply Yue's change in dev and run `npm run build` in `fx-core`, make sure static options contains all values. Only dynamic options depends on feature flag:
![image](https://github.com/OfficeDev/TeamsFx/assets/1658418/70386fb8-19fa-432f-8e35-19db3ba3c809)

Reply:
DONE